src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -32,6 +33,27 @@ static void Main()
         }
 
         [Fact]
+        [WorkItem(19013, "https://github.com/dotnet/roslyn/issues/19013")]
+        public void TestCSharp7Cascade()
+        {
+            string source = @"
+using System.Threading;
+using System.Threading.Tasks;
+
+class C
+{
+    async Task M(CancellationToken t = default) { await Task.Delay(0); }
+}
+";
+            var comp = CreateCompilationWithMscorlib46(source);
+            comp.VerifyDiagnostics(
+                // (7,40): error CS8107: Feature 'default literal' is not available in C# 7. Please use language version 7.1 or greater.
+                //     async Task M(CancellationToken t = default) { await Task.Delay(0); }
+                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion7, "default").WithArguments("default literal", "7.1").WithLocation(7, 40)
```

Comment:
> if (expr.HasAnyErrors)

It sounds like we should change implementation of HasAnyErrors helper to ignore any errors that a caused by language version mismatch.